src/librustc/diagnostics.rs

```diff
@@ -370,24 +747,25 @@ type X = u32<i32>; // error: type parameters are not allowed on this type
 
 Please check that you used the correct type and recheck its definition. Perhaps
 it doesn't need the type parameter.
+
 Example:
 
 ```
-type X = u32; // ok!
+type X = u32; // this compiles
 ```
 "##,
 
 E0110: r##"
-You tried to give a lifetime parameter to a type which doesn't need it.
-Erroneous code example:
```

Comment:
They're okay, a tad awkward though :smile:  "Example of erroneous code:" might be better too.